atspi-common/src/macros.rs

```diff
@@ -432,41 +450,49 @@ macro_rules! zbus_message_qtspi_test_case {
       #[cfg(feature = "zbus")]
      #[test]
     fn zbus_message_conversion_qtspi() {
-      // in the case that the body type is EventBodyOwned, we need to also check successful
+		use crate::events::EventTypeProperties;
+		use crate::events::MessageConversion;
+
+		// in the case that the body type is EventBodyOwned, we need to also check successful
       // conversion from a QSPI-style body.
```

Comment:
Why were all of these missing proper formatting before? Or is the diff messed up due to a rebasing problem?

Reply:
Macros are always messy. For some reason unknown to me, cargo-fmt does not touch macros.
I just either clean them up manually a bit - or when my fmt-OCD really kicks in - I go somewhere the macro is used, expand it there, have it formatted, copy it back and put the `$` variables back in place.